tests/ghc.nix

```diff
@@ -1,11 +1,11 @@
-{ pkgs ? import <nixpkgs> {} }:
+{ pkgs ? import ../nixpkgs.nix {} }:
 
 with pkgs;
 
 let haskellPackages = pkgs.haskell.packages.ghc822.override {
       overrides = with pkgs.haskell.lib; self: super: rec {
-        lens-labels = super.lens-labels_0_2_0_1;
```

Comment:
Why not to delete the whole override if we don't actually change anything with it?

Reply:
Good point. Wasn't thinking!